docs/pages/installation/_index.md

```diff
@@ -5,20 +5,252 @@ weight: 2
+* Have a look to the [pre-install section](Advanced/_index.md), that presents some common patterns used to expose and interconnect clusters when using Liqo.
 
-| K8s platforms/Service                                         | Supported                      | Notes                                  |
-| ------------------------------------------------------------- | ------------------------------ | -------------------------------------- |
-| [Azure Kubernetes Service (AKS)](./platforms/aks)             | Yes                            |                                        |
-| [Google Kubernetes Engine (GKE)](./platforms/gke)             | Yes                            |                                        |
-| [K3s](./platforms/k3s)                                        | Yes                            |                                        |
-| [Kubernetes with Kubeadm](./platforms/k8s)                    | Yes                            |                                        |
-| [AWS Elastic Kubernetes Service (EKS)](./platforms/k8s)       | In Progress                    |                                        |
+### Quick Install
 
-#### Peer your clusters
+#### Liqoctl 
 
-When you have installed Liqo on your clusters, you can decide to peer them to offload your applications on a different cluster as documented in [usage section](/usage).
\ No newline at end of file
+Liqoctl is the swiss-knife CLI tool to install and manager Liqo. 
+It is strongly recommended installing Liqo using Liqoctl since it provides customizations for supported providers.
+
+To install Liqoctl, first, you have to set the architecture and OS of your host:
+
+```bash
+#(Draft)
+OS=linux # possible values: linux,windows,darwin
+ARCH=amd64 # possible values: amd64,arm64 
+```
+
+Then, you should execute the following commands the latest version of liqoctl:
+```
+LATEST_RELEASE=$(curl -L -s -H 'Accept: application/json' https://github.com/liqotech/liqo/releases/latest)
+LATEST_VERSION=$(echo $LATEST_RELEASE | sed -e 's/.*"tag_name":"\([^"]*\)".*/\1/')
+curl --fail -LSO "https://github.com/liqotech/liqo/releases/download/${LATEST_VERSION}/liqoctl-${OS}-${ARCH}" && chmod +x liqoctl && sudo cp liqoctl-${OS}-${ARCH} /usr/bin/liqoctl
+```
+
+Obviously, you can obtain the same result by visiting the [Liqo releases](https://github.com/liqotech/liqo/releases/) page on our project.
+
+#### Pre-Requirements
+
+{{% notice note %}}
+Liqo only supports Kubernetes >= 1.19.0.
+{{% /notice %}}
+
+According to your cluster provider, you may have to perform simple steps before triggering the installation process:
+
+{{< tabs >}}
+{{% tab name="K8s (Kubeadm)" %}}
+
+##### Supported CNIs
+
+Liqo supports K8s clusters using the following CNIs:
+* [Weave](https://github.com/weaveworks/weave)
+* [Flannel](https://github.com/coreos/flannel)
+* [Canal](https://docs.projectcalico.org/getting-started/kubernetes/flannel/flannel)
+* [Calico](https://www.projectcalico.org/)
```

Comment:
Is it still necessary to adapt the calico configuration to avoid networking issues?

Reply:
Good catch.